src/web/app/components/datepicker/datepicker-formatter.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
+import { DatePickerFormatter } from './datepicker-formatter';
+
+describe('DatepickerFormatter', () => {
+    let formatter : DatePickerFormatter;
+
+    beforeEach(() => {
+        formatter = new DatePickerFormatter();
+    });
+
+    it('should create an instance', () => {
+        expect(formatter).toBeTruthy();
+    });
+
+    it('should return an empty string if date is null', () => {
+        // I used any as the type here as it is the only way to pass in null since
+        // null is not compatible with NgbDateStruct
```

Comment:
good to explain why, but let's change the wording a little
```suggestion
        // Any is used as the type for date here as it is the only way to pass in null to NgbDateStruct
```